src/main/java/uk/gov/hmcts/darts/event/EventTypeRepository.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.darts.event;
```

Comment:
Should this be under `package uk.gov.hmcts.darts.common.repository` to be close to `EventEntity`?